bittensor/_dataset/__init__.py

```diff
@@ -40,13 +40,21 @@ class dataset:
     def __new__(
             cls,
             config: 'bittensor.config' = None,
-            block_size: int = None,
+            block_size: int = 500,
```

Comment:
This needs to be larger; i'm missing tokens with seq_length = 256. 5* sequence length is a good rule of thumb, try 1500?